packages/cli/test/unit/commands/build/index.test.ts

```diff
@@ -407,6 +407,41 @@ describe('build', () => {
     }
   });
 
+  it('should pull "production" env vars with `--target production`', async () => {
```

Comment:
Nice test 👍 

**non-blocking:** Do we have anything in place to test this with a custom environment target? Would be nice to get those in place but seems like we may not have all the pieces for that yet